tooling/css/src/styles/globals.css

```
@@ -52,7 +52,7 @@
    --sidebar-border: oklch(0.9276 0.0058 264.5313);
    --sidebar-ring: oklch(0.7227 0.192 149.5793);
    --font-sans: Sora, ui-sans-serif, system-ui, -apple-system, sans-serif;
-   --font-serif: "Instrument Serif", ui-serif, Georgia, serif;
+   --font-serif: Lora, ui-serif, Georgia, serif;
```

Stylelint (17.11.0)

[error] Expected "Lora" to be "lora" (value-keyword-case)

(value-keyword-case)